trillian/ctfe/ct_server/main.go

```diff
@@ -65,10 +65,21 @@ func main() {
 		ctfe.MaxGetEntriesAllowed = *maxGetEntries
 	}
 
-	// Get log config from file before we start.
-	cfg, err := ctfe.LogConfigFromFile(*logConfig)
+	var cfg *configpb.LogMultiConfig
+	var beMap map[string]*configpb.LogBackend
+	var err error
+	// Get log config from file before we start. This is a different proto
+	// type if we're using a multi backend configuration (no rpcBackend set
+	// in flags). The single backend config is converted to a multi config so
+	// they can be treated the same.
+	if len(*rpcBackend) > 0 {
```

Comment:
Could do with an update to the help for `--log_rpc_server` or `--log_config` to hint at this.

(While you're there, I think the `--log_config` text is outdated as it still mentions JSON)

Reply:
Done.